Documentation/Products/UmbracoForms/Developer/Extending/Adding-a-Fieldtype.md

```diff
@@ -2,25 +2,34 @@
 
 *This builds on the "[adding a type to the provider model](Adding-a-Type.md)" chapter*
 
-Add a new class to the visual studio solution and make it inherit from Umbraco.Forms.Core.FieldType and override the Editor property.
+Add a new class to the Visual Studio solution with a using directive using `using Umbraco.Forms.Core;` and make it inherit from Umbraco.Forms.Core.FieldType with an override of the Editor property.
```

Comment:
This is also a bit confusing. Perhaps: "Add a new class to your Visual Studio solution called 'CustomFormDataType', ensure it inherits from Umbraco.Forms.Core.FieldType and create an override of the Editor property. 

Might need to start the whole thing off with, As an example we'll create a new FieldType called ....